src/components/TasksTable/index.jsx

```diff
@@ -344,6 +404,43 @@ export default class TasksTable extends Component {
             onHeaderClick={this.handleHeaderClick}
           />
         </div>
+        <Drawer
+          anchor="right"
+          open={drawerOpen}
+          onClose={this.handleDrawerClose}
+          classes={{ paper: classes.drawerPaper }}>
+          <Fragment>
+            <IconButton
+              className={classes.drawerCloseButton}
+              onClick={this.handleDrawerClose}>
+              <CloseIcon />
+            </IconButton>
+            {drawerItem && (
+              <List>
+                <ListItem>
+                  <ListItemText
+                    primary="Summary"
+                    secondary={drawerItem.summary}
+                  />
+                </ListItem>
+                {drawerItem.description && (
+                  <ListItem>
+                    <ListItemText
+                      primary="Description"
+                      secondary={drawerItem.description}
+                    />
+                  </ListItem>
+                )}
+                <ListItem>
+                  <ListItemText
+                    primary="Suggestion"
```

Comment:
Rename to "Are you interested?" and remove the "Are you suggested?" sentence from the suggestion content.

Reply:
By the way, this still needs to be done :)